dvc/command/experiments/pull.py

```diff
@@ -87,6 +87,9 @@ def add_parser(experiments_subparsers, parent_parser):
         metavar="<git_remote>",
     )
     experiments_pull_parser.add_argument(
-        "experiment", help="Experiment to pull.", metavar="<experiment>"
+        "experiment",
+        nargs="+",
+        help="Experiments to pull.",
+        metavar="<experiment>",
```

Comment:
Question. Is it normal that the help output now uses `<metavars>` ? I.e. is it necessary here?

```
positional arguments:
  <git_remote>          Git remote name or Git URL.
  <experiment>          Experiments to pull.
```

Most argument lists don't use `<>` last time I checked.

Reply:
Also, both the arg and metavar are in singular ("experiment") but the help text uses plural ("Experiments").

Reply:
Cc @dberenbaum rel. https://github.com/iterative/dvc.org/pull/3192#discussion_r834284360

Reply:
@jorgeorpinel , I tried to removed line `metavar="<experiment>",` the output will be
```
$ dvc exp pull --help
usage: dvc experiments pull [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f] [--no-cache] [-r <name>] [-j <number>] [--run-cache] git_remote [experiment [experiment ...]]
positional arguments:
  git_remote            Git remote name or Git URL.
  experiment            Experiments to pull.
...
```
By comparison the current version is:
```
$ dvc exp pull --help
usage: dvc experiments pull [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f] [--no-cache] [-r <name>] [-j <number>] [--run-cache] <git_remote> [<experiment> [<experiment> ...]]

positional arguments:
  <git_remote>          Git remote name or Git URL.
  <experiment>          Experiments to pull.
```
Similiar for the `exp push`

Reply:
Up to you guys, I was just pointing it out as it seems a little inconsistent with other commands' help output.